openshift/assisted-service
# MGMT-9984: Support iPXE in converged ZTP flow

Set the PXE artifacts on the preprovisioningimage CR in the
preprovisioningimage controller will set in case the
preprovisioningimage accepts Initrd image format (and doesn't accept ISO)

## List all the issues related to this PR

https://issues.redhat.com/browse/MGMT-9984


## What environments does this code impact?


## How was this code tested?

- Deploy a dev-scripts cluster with extra hosts.
- Create BMH for extra hosts using `redfish` protocol (not `redfish-virtualmedia`)
- Observe artifact urls in preprovisioning image




## Checklist


## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?
